server/server.go

```diff
@@ -192,9 +192,11 @@ func (s *Server) HTTPHandler() http.Handler {
 	)
 
 	if s.prometheusProxyEnabled() {
+		// Only proxy requests to the Prometheus API, not the UI.
+		prometheusProxyAPIPath := prometheusProxyEndpoint + "/api/"
```

Comment:
Does this actually work? Looking at this: https://github.com/openshift/console/blob/eae371ba5f35f1b3b295625e5e5b4f2de4fdd637/pkg/proxy/proxy.go#L112

It looks to me that the resulting request received by Prometheus is going to be something along the lines of `/api/api/v1/query`.